<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Default Resource Type:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxDefaultResource</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Picture Editor Style:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxPicEditStyle</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QComboBox" name="comboBoxPicEditStyle">
         <item>
          <property name="text">
           <string>One Window</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Two Windows</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBoxExtractLogic">
         <property name="title">
          <string>Extract Logic As:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButtonExtractAsText">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Text</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupExtractLogicAs</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonExtractAsBin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Binary</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupExtractLogicAs</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxDefaultResource">
         <item>
          <property name="text">
           <string>Logic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Picture</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>View</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sound</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDirectories">
      <attribute name="title">
       <string>Directories</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButtonTemplateDirBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditTemplateDir"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditHelpDir"/>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Logic Source Directory</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditGameDirSrc"/>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButtonFullPath">
            <property name="text">
             <string>Full Path:</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupSrcDir</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditFullPathSrc"/>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButtonGameDir">
            <property name="text">
             <string>[GameDir]/</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupSrcDir</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButtonSrcDirBrowse">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Template:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditTemplateDir</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButtonHelpDirBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Help:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditHelpDir</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLogicEditor">
      <attribute name="title">
       <string>Logic Editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBoxShowAllMessages">
         <property name="text">
          <string>Show all messages at end (not just unused ones)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxShowElseAsGoto">
         <property name="text">
          <string>Show all elses as gotos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxShowSpecialSyntax">
         <property name="text">
          <string>Show special syntax (e.g. v30=4)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInterpreter">
      <attribute name="title">
       <string>Interpreter</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButtonInterpPathBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Arguments:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditInterpArgs</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditInterpArgs"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditInterpPath"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Program:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxInterpreter</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxInterpreter">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>ScummVM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NAGI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sarien</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other...</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Path:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditInterpPath</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Use the string %1 in the argument list to specify the game directory, if required.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupExtractLogicAs"/>
  <buttongroup name="buttonGroupSrcDir"/>
 </buttongroups>
</ui>
